modules/administration/pages/subscription-matching.adoc

```diff
@@ -1,153 +1,61 @@
 
-Selecting menu:Main Menu[Audit > Subscription Matching] from the left navigation menu will provide you with an overview of all results generated by the Subscription Matcher.
+The [guimenu]``Messages`` tab shows any errors that occurred during the matching process.
 
+You can also download the reports in .csv format, or access them from that command prompt in the [path]``/var/lib/spacewalk/subscription-matcher/`` directory.
 
-[IMPORTANT]
-.Subscription Matcher Accuracy
-====
-This tool's goal is to help provide visual coverage on current subscription use and support reporting.
-The Subscription Matcher is excellent at matching systems and products registered with {productname}, however any systems, products or environments which are not found in the database will remain unmatched.
-This tool is not intended to act as a replacement for auditing.
-Auditing should always take precedence over subscription matching.
-====
-
-
-
-== Subscriptions
-
-The Subscription Matching overview provides subscription part numbers, product descriptions, policies, matched total subscriptions used and remaining, and the start and end dates of subscriptions.
-
-.Subscription Matching Overview
-image::smatching-overview.png[scaledwidth=80%]
-
-Part Number::
-Identifier of a particular product
-
-Description::
-Name of a particular product
-
-Policy::
-Kind of the subscription of this product
-
-Matched/Total::
-** {empty}
-+
-
-.Fully Matched
-If the total amounts of a subscription are fully matched, the quantity column value is highlighted with a yellow warning triangle: image:fa-warning.svg[Warning,scaledwidth=0.9em]
-** {empty}
-+
-
-.Susbscritions about to Expire
-When a subscription will expire within less than 3 months, the record is highlighted.
-** {empty}
-+
-
-.Expired Subscriptions
-If a subscription is expired, the record for it is faded.
-
-Start Date::
-Start date of the subscription
-
-End Date::
-End date of the subscription
-
-== Subscription Matcher Reports
-
-{productname} automatically generates up-to-date nightly status reports by matching your {suse} subscriptions with all your registered systems.
-These reports are stored in [path]``/var/lib/spacewalk/subscription-matcher`` and provided in CSV format.
-These CSV files may be opened with a spreadsheet application such as LibreOffice Calc.
-
-If you want to schedule these reports to be produced at different times, or at a certain frequency or schedule a one time completion you can perform this task by editing the Taskomatic settings for the gatherer-matcher located in the {webui} at menu:Main Menu[Admin > Task Schedules > gatherer-matcher-default].
-
-image::audit_subscription_matching_gatherer_matcher_default.png[scaledwidth=80%]
+By default, the subscription matcher runs daily, at midnight.
+To change this, navigate to menu:Admin[Task Schedules] and click ``gatherer-matcher-default``.
+Change the schedule as required, and click btn:``Update Schedule``.
```

Comment:
I'm not sure whether this button markup is also valid asciidoc decoration.  I think our default markup is btn:[xxx].
Broken by design.

Reply:
Oh, that's just me getting confused. You're right :)